src/Kernels.h

```diff
@@ -1,6 +1,16 @@
 #ifndef COVIDSIM_KERNELS_H_INCLUDED_
 #define COVIDSIM_KERNELS_H_INCLUDED_
 
+enum class Kernel_Type {
+    KERNEL_EXPONENTIAL = 1,
+    KERNEL_POWER = 2,
+    KERNEL_GUSSIAN = 3,
+    KERNEL_STEP = 4,
+    KERNEL_POWERB = 5, // figure out a better name, power b isn't descriptive
+    KERNEL_POWERUS = 6,
+    KERNEL_EXPONENTIAL_POWER = 7,
+};
```

Comment:
If using `enum class` no need to put `KERNEL_` as a prefix to each value as we have to use the class name to qualify the value.